### PR TITLE
Add Pulsar Community Weekly Update - 2020-09-19~ 2020-09-25

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    https://github.com/apache/pulsar/pull/7993(@[murong00](https://github.com/murong00))
+
+- [Topic Policy] Support setting, getting, and removing the subscription dispatch rate on the topic level.
+
+    https://github.com/apache/pulsar/pull/8087(@[hangc0276](https://github.com/hangc0276))
+
+- [Topic Policy] Support setting, getting, and removing the subscribe rate on the topic level.
+
+    https://github.com/apache/pulsar/pull/7991(@[hangc0276](https://github.com/hangc0276))
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-25-pulsar-weekly.md:69]: duplicated?

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+- [Client] Support dynamically loading the Truststore certificate through the  input stream.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-25-pulsar-weekly.md:71]:
```suggestion
- [Client] Support dynamically loading the Truststore certificate through the input stream.
```

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+	
+    https://github.com/apache/pulsar/pull/7442(@[rdhabalia](https://github.com/rdhabalia))
+
+- [Broker] Replace `IdentityHashMap` with `set -- Collections.newSetFromMap(new ConcurrentHashMap<>())` to hold the reference of producers/consumers .
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-25-pulsar-weekly.md:75]: remove the redundant space at the end.

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+- [Functions] Fix Pulsar Functions threaded integration tests.
+
+   https://github.com/apache/pulsar/pull/8118(@[jerrypeng](https://github.com/jerrypeng))
+
+- [Client] Use the bytes of messages instead the number of messages for the producer to pend messages.
+
+    https://github.com/apache/pulsar/pull/7957(@[Renkai](https://github.com/Renkai))
+
+- [Client] Fix client lookup hangs when the broker restarts.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-25-pulsar-weekly.md:89]: replace "hang" with another word

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+This is the Pulsar community weekly update for 2020-09-19 ~ 2020-09-25, with updates on Pulsar client, broker, Pulsar SQL, Pulsar Functions, and so on.
+
+### Development
+
+- [Broker] Use SNI for the TLS-enabled Pulsar Java broker client.
+
+    https://github.com/apache/pulsar/pull/8117(@[racorn](https://github.com/racorn))
+
+- [Pulsar SQL] Replace `com.ning.asynchttpclient` with `org.asynchttpclient`.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-25-pulsar-weekly.md:19]: what's the goal of replacement? Shall we add it?

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md (hunk 1 above)
Comment (Huanli-Meng) [reply content/2020/2020-09/2020-09-25-pulsar-weekly.md:69]: no, these are two kinds of setting, one is for dispatch rate and the other is the subscription rate

File: content/2020/2020-09/2020-09-25-pulsar-weekly.md (hunk 4 above)
Comment (Huanli-Meng) [reply content/2020/2020-09/2020-09-25-pulsar-weekly.md:89]: Updated as : Fix the issue that the the client lookup hangs when the broker restarts.

**Comment by Huanli-Meng** [comment]
@Jennifer88huang Thanks a lot for your comments. Doc is updated based on your comments. Please help double check it. Thanks.

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**